scripts/scripts.js

```diff
@@ -78,7 +78,11 @@ async function loadFonts() {
 // eslint-disable-next-line no-unused-vars
 function buildAutoBlocks(main) {
   try {
-    // TODO: add auto block, if needed
+    const pictureElement = document.querySelector('.default-content-wrapper > p > picture');
```

Comment:
1. Instead of document.querySelector, use main.querySelector
2. Move the logic to a method `adjustAutoImages()` and call the method here.
3. Change class name from `image-container` to `auto-image-container`